comps/MyLayout.js

```diff
@@ -0,0 +1,16 @@
+import Header from "./Header";
+
+const layoutStyle = {
+  margin: 20,
+  padding: 20,
+  border: "1px solid #DDD"
+};
+
+const Layout = props => (
+  <div style={layoutStyle}>
+    <Header />
+    {props.children}
+  </div>
+);
```

Comment:
>can you explain the {props.children} under MyLayout.js?
>
>What are props again and how are they passed down?

This "fat arrow" function is using a new-ish "shorthand syntax" for writing functions, the long form version would be like this:

```js
function Layout(props) {
  return (
    <div style={layoutStyle}>
      <Header />
      {props.children}
    </div>
  );
}
```

Hopefully this makes it a little clearer that "props" are just an argument to the "Layout" function. In the shorthand syntax, if your function only has one argument you can omit the parenthesis … and if your function just returns something you can omit the curly braces. So `const Layout = (props) => { return <div />; };` is the same as `const Layout = props => <div />;`. In this particular case the return value takes up more than one line, so you've got to wrap it in parenthesis to tell the runtime that it's all one value.